src/main/groovy/edu/oregonstate/mist/musicapi/resources/AlbumResource.groovy

```diff
@@ -0,0 +1,58 @@
+package edu.oregonstate.mist.musicapi.resources
+
+import edu.oregonstate.mist.api.AuthenticatedUser
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.musicapi.core.Album
+import io.dropwizard.auth.Auth
+import javax.ws.rs.GET
+import javax.ws.rs.POST
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.Consumes
+import javax.ws.rs.core.Response
+import javax.ws.rs.core.Response.ResponseBuilder
+import org.skife.jdbi.v2.DBI
+
+@Path('/album/{id}')
```

Comment:
Because id is declared as an int in the methods below (`@PathParam("id") int id`), dropwizard already knows id should be numeric and checks accordingly.